types/mailchimp__mailchimp_transactional/index.d.ts

```diff
@@ -0,0 +1,177 @@
+// Type definitions for @mailchimp/mailchimp_transactional 1.0
+// Project: https://github.com/mailchimp/mailchimp-transactional-node
+// Definitions by: Pieter Scheffers <https://github.com/PieterScheffers>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.5
```

Comment:
```suggestion
```
is this required for some reason? (version specific feature, etc). This can limit interop with other types

Reply:
No this is not required. At first I thought it was required. I will remove it!